components/automate-backend-elasticsidecar/habitat/bin/elastic_sidecar.rb

```diff
@@ -20,166 +20,212 @@ def logger
-  #      /hab/svc/automate-ha-elasticsidecar/config/securityconfig/config.yml
-  #      /hab/svc/automate-ha-elasticsidecar/config/securityconfig/roles.yml
-  #    ], config['securityconfig_path']
-  #  end
+    def setup_templates
+      unless Dir.exist?(config['securityconfig_path'])
+        FileUtils.mkdir_p(config['securityconfig_path'])
+      end
+      FileUtils.cp %w[
+        /hab/svc/automate-ha-elasticsidecar/config/securityconfig/action_groups.yml
+        /hab/svc/automate-ha-elasticsidecar/config/securityconfig/config.yml
+        /hab/svc/automate-ha-elasticsidecar/config/securityconfig/roles.yml
+        /hab/svc/automate-ha-elasticsidecar/config/securityconfig/tenants.yml
+        /hab/svc/automate-ha-elasticsidecar/config/securityconfig/nodes_dn.yml
+        /hab/svc/automate-ha-elasticsidecar/config/securityconfig/whitelist.yml
+      ], config['securityconfig_path']
+    end
 #
     def wait
       logger.debug "Sleeping 60s"
       sleep 60
     end
 #
-  #  def run
-  #    loop do
-  #      response = test_authentication(config['admin_username'], config['admin_password'])
-  #      if response.eql? 'error'
-  #        wait
-  #        next
-  #      elsif response.code == 200
-  #        logger.debug "auth successful for #{config['admin_username']}"
-  #      end
+    def run
+      loop do
+        response = test_authentication(config['admin_username'], config['admin_password'])
+        if response.eql? 'error'
+          wait
+          next
+        elsif response.code == 200
+          logger.debug "auth successful for #{config['admin_username']}"
+        end
 #
-  #      case response.code
-  #      when 200
-  #        logger.debug 'Authentication successful, doing nothing'
-  #      when 401, 403
-  #        logger.warn 'Authentication failed, inserting credentials'
-  #        rotate_credentials
-  #      when 503
-  #        if response.body.to_s.eql? 'Open Distro not initialized'
-  #          logger.warn 'Open Distro appears to not be setup, inserting credentials'
-  #          rotate_credentials
-  #        elsif response.body.to_s.valid_json?
-  #          logger.warn 'Auth successful, but elasticsearch appears to be broken, doing nothing.'
-  #        else
-  #          logger.warn 'Elasticsearch returned 503 error with unexpected message'
-  #          logger.warn response.body
-  #        end
-  #      else
-  #        logger.error "Elasticsearch returned #{response.code} code with unexpected message"
-  #        logger.error response.body
-  #      end
-  #      wait
-  #    end
-  #  end
+        case response.code
+        when 200
+          logger.debug 'Authentication successful, doing nothing'
+        when 401, 403
+          logger.warn 'Authentication failed, inserting credentials'
+          rotate_credentials
+        when 503
+          if response.body.to_s.eql? 'Open Distro not initialized'
+            logger.warn 'Open Distro appears to not be setup, inserting credentials'
```

Comment:
please update the this log, as we are not using the open distro package any more